app/main/db/queries.py

```diff
@@ -70,3 +70,42 @@ def fuzzy_search(query_string):
     except exc.SQLAlchemyError as e:
         print("Failed to return results from database with error : " + str(e))
     return results
+
+
+def browse_view():
```

Comment:
can we rename this to?
```suggestion
def get_file_data_grouped_by_transferring_body_and_series():
```

Reply:
function renamed as requested